base/src/main/kotlin/net/corda/v5/base/types/LayeredPropertyMap.kt

```diff
@@ -7,57 +7,102 @@ package net.corda.v5.base.types
  * The layered property map provides simple conversions to a possibly complex objects which can use several keys in
  * dot-notation. Take as an example of the map:
  *
- * "corda.name" to "CN=me, O=R3, L=Dublin, C=Ireland",
- * "corda.sessionKey" to "ABCDEF...",
- * "corda.endpoints.0.url" to "localhost",
- * "corda.endpoints.0.protocolVersion" to "1",
- * "corda.endpoints.1.url" to "localhost",
- * "corda.endpoints.1.protocolVersion" to "2"
+ * - "corda.name" to "CN=me, O=R3, L=Dublin, C=Ireland",
+ * - "corda.sessionKey" to "ABCDEF...",
+ * - "corda.endpoints.0.url" to "localhost",
+ * - "corda.endpoints.0.protocolVersion" to "1",
+ * - "corda.endpoints.1.url" to "localhost",
+ * - "corda.endpoints.1.protocolVersion" to "2"
  *
  * That map can be parsed into:
  * - MemberX500Name using parse("corda.name", MemberX500Name::class.java)
  * - session PublicKey using parse("corda.sessionKey", PublicKey::class.java)
  * - list of endpoints using parseList("corda.endpoints", EndpointInfo::class.java)
  *
+ * Example usages:
+ *
+ * ```java
+ * Set<Map.Entry<String, String>> entries = propertyMap.getEntries();
+ * String groupId = propertyMap.parse("corda.groupId", String.class);
+ * Instant modifiedTime = propertyMap.parseOrNull("corda.modifiedTime", Instant.class);
+ * Set<String> additionalInformation = propertyMap.parseSet("additional.names", String.class);
+ * List<EndpointInfo> endpoints = propertyMap.parseList("corda.endpoints", EndpointInfo.class);
+ * ```
+ *
+ * ```kotlin
+ * val entries = propertyMap.entries
+ * val groupId = propertyMap.parse("corda.groupId", String::class.java)
+ * val modifiedTime = propertyMap.parseOrNull("corda.modifiedTime", Instant::class.java)
+ * val additionalInformation = propertyMap.parseSet("additional.names", String::class.java)
+ * val endpoints = propertyMap.parseList("corda.endpoints", EndpointInfo::class.java)
+ * ```
+ *
  * The default implementation of the [LayeredPropertyMap] is extendable by supplying implementations of custom
  * converters using OSGi. Out of box it supports conversion to simple types like Int, Boolean,
- * as well as MemberX500Name.
+ * as well as [MemberX500Name].
+ *
+ * @property entries Returns [Set] of all entries in the underlying map.
+ *
+ * @author Alexey Kadyrov
```

Comment:
I think we are not using @author tag

Reply:
As i understood from the guidelines, we should use them:
 https://r3-cev.atlassian.net/wiki/spaces/CB/pages/4106027171/Guidance+for+completing+Corda-API+KDocs

Reply:
There was a discussion for that attribute in the doc - it's useless in the context of R3, e.g. I'm not an author of the MemberX500Name as it was ported from C4 codebase with some minor modifications.

Reply:
@mattbradburyr3 As discussed, I think we should remove @author as a requirement?